config/config.yaml

```diff
@@ -162,8 +162,7 @@ defaults:
     rg: hcp-underlay-{{ .ctx.regionShort }}-imagesync
     acrRG: global
     environmentName: aro-hcp-image-sync
-    # secretsToSync: "dev_global_imagesync_bearer-secret.enc;dev_global_imagesync_component-sync-pull-secret.enc;"
-    secretsToSync: ""
+    secretsToSync: ewogICAgImJlYXJlci1zZWNyZXQiOiAiZGV2X2dsb2JhbF9pbWFnZXN5bmNfYmVhcmVyLXNlY3JldC5lbmMiLAogICAgImNvbXBvbmVudC1zeW5jLXB1bGwtc2VjcmV0IjogImRldl9nbG9iYWxfaW1hZ2VzeW5jX2NvbXBvbmVudC1zeW5jLXB1bGwtc2VjcmV0LmVuYyIKfQ==
```

Comment:
i would prefer to not have unreadable config values.

i wonder if we can reduce the need for configuration a lot. e.g. group secrets to sync to an env/KV in directory structures and name the files exactly as the secrets will be named in the KV. the config could be something as simple as "what is the dir and kv to process"